lib/pip.js

```diff
@@ -210,7 +219,7 @@ function installRequirements(targetFolder, serverless, options) {
           ])
         );
       }
-      pipCmd = ['/bin/bash', '-c', '"' + commands.join(' && ') + '"'];
+      pipCmd = ['/bin/bash', '-c', '"' + commands.join(' ; ') + '"'];
```

Comment:
I think this is bad... this will have the side effect of if any command fails it won't (necessarily) detect it upstream, the &&'s are important in this context.  Was there a reason for this necessary for this feature?